addons/account-settings-capitalize/addon.json

```diff
@@ -7,6 +7,6 @@
       "matches": ["https://scratch.mit.edu/*"]
     }
   ],
-  "tags": ["easterEgg", "community"],
```

Comment:
No `easterEgg` tag for this addon anymore?

Reply:
No, since it was really never an easter egg. Just a minor addon